<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportTabRange</class>
 <widget class="QWidget" name="ReportTabRange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chart Tab</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string>&lt;p&gt;On this tab, you configure the chart drawn for this report.&lt;/p&gt;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
   <item>
    <widget class="QGroupBox" name="dateRangeBox">
     <property name="title">
      <string>Date range</string>
     </property>
     <layout class="QGridLayout" name="dateRangeGrid">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Ticks</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KComboBox" name="m_comboColumns">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>&lt;p&gt;Choose how large of a time period each column should encompass&lt;/p&gt;</string>
          </property>
          <item>
           <property name="text">
            <string comment="@item the columns will display daily data">Daily</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string comment="@item the columns will display weekly data">Weekly</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string comment="@item the columns will display monthly data">Monthly</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string comment="@item the columns will display bi-monthly data">Bi-Monthly</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string comment="@item the columns will display quarterly data">Quarterly</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string comment="@item the columns will display yearly data">Yearly</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="dataRangeBox">
     <property name="title">
      <string>Data range</string>
     </property>
     <layout class="QGridLayout" name="dataRangeLayout">
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <item row="3" column="0">
       <widget class="QLabel" name="lblDataRangeStart">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string comment="From (Data)">From</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblDataMajorTick">
        <property name="text">
         <string>Major tick length</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="m_dataMinorTick"/>
      </item>
      <item row="8" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="m_dataRangeEnd"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Range</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="m_dataLock">
        <item>
         <property name="text">
          <string>Automatic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>User defined</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lblDataMinorTick">
        <property name="text">
         <string>Minor tick length</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="m_dataMajorTick"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblDataRangeEnd">
        <property name="text">
         <string comment="To (Data)">To</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="m_dataRangeStart"/>
      </item>
      <item row="7" column="1">
       <widget class="QSpinBox" name="m_yLabelsPrecision">
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="lblLabelsPrecision">
        <property name="text">
         <string>Labels precision</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_dataLock</tabstop>
  <tabstop>m_dataRangeStart</tabstop>
  <tabstop>m_dataRangeEnd</tabstop>
  <tabstop>m_dataMajorTick</tabstop>
  <tabstop>m_dataMinorTick</tabstop>
  <tabstop>m_yLabelsPrecision</tabstop>
  <tabstop>m_comboColumns</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
